api/src/routers/topics.js

```diff
@@ -5,4 +5,6 @@ const router = express.Router();
 
 router.post('/', createOrUpdateTopic);
 
+router.post('/edit/', createOrUpdateTopic);
```

Comment:
split up this function into create or update. Also technically this should be a router.put function